src/Search_Replace_Command.php

```diff
@@ -640,6 +648,7 @@ private static function get_columns( $table ) {
 		$text_columns    = array();
 		$all_columns     = array();
 		$suppress_errors = $wpdb->suppress_errors();
+		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared -- escaped through self::esc_sql_ident
 		$results         = $wpdb->get_results( "DESCRIBE $table_sql" );
```

Comment:
As there is now a comment breaking the assignment list, the whitespace before the `=` here needs to be reduced to one space.